### DISCUSSION TITLE
How can i do if i forgot password, no email STMP setup?

### DISCUSSION THREAD

**Discussion by tracer8** [requarks/wiki#3846] [Help / Questions] [answered]
Hi,
I'm setup Wiki.js server. And i forgot password. But no email STMP setup before. So i can't use "Forgot Pass" feature.
Is there any way can i reset my password?

**Comment by Phil-hostwiki** [comment, marked as answer]
1. Use https://bcrypt-generator.com to generate a bcrypt hash of your new password with 12 rounds.

2. I'm assuming you are using docker. 
  SSH into your server and run
 `docker exec db psql -U wiki -d wiki -c "UPDATE users SET password = 'HASH-PASSWORD' WHERE email = 'YOUR-EMAIL';"`

**Comment by tracer8** [comment]
Thank you so much!
